client/src/elements/molecules/item-list-entry.css

```diff
@@ -32,24 +24,110 @@ item-list-entry > a {
   width: 100%;
 }
 
-item-list-entry a > span {
+item-list-entry a > span:not(.item-list-entry__state-tag) {
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  flex-grow: 1;
+  flex-basis: 0;
+}
+
+.item-list-entry__state-tag {
+  height: 20px;
+  line-height: 20px !important;
+  min-width: calc(var(--q-space-base) * 7);
+  border-radius: 10px;
+  margin-right: calc(var(--q-space-base) * 2);
+  text-align: center;
+}
+
+.item-list-entry__state-tag--active {
+  background-color: var(--q-color-primary-2);
+  color: var(--q-color-white);
+}
+
+.item-list-entry__state-tag--inactive {
+  background-color: var(--q-color-primary-1);
+  color: var(--q-color-primary-2);
+}
+
+.item-list-entry__options-dropdown-button {
+  width: 18px;
+  margin-right: var(--q-space-base);
+}
+
+.item-list-entry__options-dropdown-button icon-button {
+  height: 28px;
 }
 
-item-list-entry a > span:nth-of-type(1) {
-  width: calc((100% - (2 * 26px)) / 10 * 4);
```

Comment:
these fixed percentage based widths have to be here afaik to have the layout not jump when items with longer titles get loaded. but there may be another solution.